app/controllers/cart_items_controller.rb

```diff
@@ -0,0 +1,34 @@
+class CartItemsController < ApplicationController
+  def index
+    if logged_in?
+      @cart_items = current_user.cart.cart_items
+    end
+  end
+
+  def new
+    @cart_item = CartItem.new
+  end
+
+  def show
+    @dish = Dish.find(params[:id])
+  end
+
+  def create
+    @cart_item = CartItem.new(cart_item_params)
+    if @cart_item.save
+      flash[:success] = "Item added to your cart!"
+      redirect_to root_path
+    else
+      flash[:danger] = 'Could not add item to the basket!'
+      redirect_to root_path
+    end
+  end
+
+  private
+
+    def cart_item_params
+      params.require(:cart_item).permit(:cart_id, :dish_id, :quantity)
```

Comment:
I think a better structure would be:
```ruby
params.require(:cart).permit(:id, :dish_id, :quantity)
```

Reply:
Are you sure about it? The cart_item is an intermediary model between Carts and Dishes, therefore the cart_id and dish_id